Makefile

```diff
@@ -133,9 +133,10 @@ test: fmt vet envtest ## Run tests.
 
 .PHONY: build
 build: fmt vet bin/vm-builder ## Build all neonvm binaries.
-	GOOS=linux go build -o bin/controller       neonvm/main.go
-	GOOS=linux go build -o bin/vxlan-controller neonvm/tools/vxlan/controller/main.go
-	GOOS=linux go build -o bin/runner           neonvm/runner/*.go
+	GOOS=linux go build -o bin/controller       neonvm-controller/cmd/main.go
+	GOOS=linux go build -o bin/vxlan-controller neonvm-vxlan-controller/cmd/main.go
+	GOOS=linux go build -o bin/daemon           neonvm-daemon/main.go
+	GOOS=linux go build -o bin/runner           neonvm-runner/cmd/main.go
```

Comment:
Are these required for this PR? (if no: let's break that out into a separate change? will be easier to discuss without derailing this PR)

Reply:
Not really but I use build target locally for fast builds and run linter

Reply:
Hm, that sounds like we should potentially improve our local setup there. I always just manually run `go build ./...` or `golangci-lint run` (approximately — the actual commands are much more convoluted, but I have them in my shell history...)

If you also think there's room for improvement, could you open an issue?

Reply:
We discussed this 1-1 — I'm ok to leave this change in the PR, but maybe we should just remove `make build` entirely (we already have `make lint`, etc.)